jujugui/static/gui/src/app/components/charmbrowser/charmbrowser.js

```diff
@@ -161,6 +162,8 @@ YUI.add('charmbrowser-component', function() {
           );
           break;
         case 'entity-details':
+          // TODO frankban: do we still really need this?
+          const id = currentState.store || `~${currentState.user}`;
```

Comment:
IIRC this was when we were supporting both the old and new id's. So I want to say, no we don't need it - but removing it on release day is probably a bad idea :D